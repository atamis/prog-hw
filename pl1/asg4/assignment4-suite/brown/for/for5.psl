deffun error() +("error", 5) in
defvar x = 0 in
deffun update()
  x++;
  print("x=");
  print(x);
  print("\n");
in {
  for(print("init\n");
      if ==(x, 1) then error() else true;
      update()) {
    print("body\n");
    print(x);
    print("\n");
  }
}
