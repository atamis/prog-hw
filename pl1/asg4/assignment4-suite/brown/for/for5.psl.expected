init
body
0
x=1
